messages/org.json

```diff
@@ -1,4 +1,10 @@
 {
-    "NotFoundOnDevHub": "The scratch org does not belong to the dev hub username %s.",
-    "NotADevHub": "The provided dev hub username %s is not a valid dev hub."
-}
\ No newline at end of file
+  "NotFoundOnDevHub": "The scratch org does not belong to the dev hub username %s.",
+  "NotADevHub": "The provided dev hub username %s is not a valid dev hub.",
+  "NoDevHubFound": "Unable to associate this scratch org with a DevHub",
+  "deleteOrgHubError": "The Dev Hub org cannot be deleted.",
+  "insufficientAccessToDelete": "You do not have the appropriate permissions to delete a scratch org. Please contact your Salesforce admin.",
+  "attemptingToDeleteExpiredOrDeleted": "Attempting to delete an expired or deleted org",
+  "sandboxDeleteFailed": "The sandbox org deletion failed with a result of %s.",
+  "sandboxProcessNotFoundByOrgId": "We can't find a SandboxProcess for the sandbox org %s."
```

Comment:
If these are error names then they should start with a capital letter and be more generically named.  Leave the details to the message.